lnwallet/rpcwallet/rpcwallet.go

```diff
@@ -154,7 +154,7 @@ func (r *RPCKeyRing) SendOutputs(inputs fn.Set[wire.OutPoint],
 		// watch-only wallet if it can map this outpoint into a coin we
 		// own. If not, then we can't continue because our wallet state
 		// is out of sync.
-		info, err := r.WalletController.FetchInputInfo(
+		info, err := r.WalletController.FetchOutpointInfo(
```

Comment:
Makes sense, `FetchInputInfo` ends up calling `FetchOutpointInfo` under the covers: https://github.com/btcsuite/btcwallet/blob/6ecae9c12fde8ed4db9e3c88241a8aeabff37724/wallet/utxos.go#L112-L123